File: GalleryApp/src/main/java/Model/Artist.java
package Model;

import java.util.ArrayList;
import java.util.List;

public class Artist {

    private String name;
    private String birthDate;
    private String birthPlace;
    private String nationality;
    private String photo;
    private List<Artwork> artworks;

    public Artist(String name, String birthDate, String birthPlace, String nationality, String photo) {
        this.name = name;
        this.birthDate = birthDate;
        this.birthPlace = birthPlace;
        this.nationality = nationality;
        this.photo = photo;
        this.artworks = new ArrayList<>();
    }


    public String getName() { return name; }
    public void setName(String name) { this.name = name; }

    public String getBirthDate() { return birthDate; }
    public void setBirthDate(String birthDate) { this.birthDate = birthDate; }

    public String getBirthPlace() { return birthPlace; }
    public void setBirthPlace(String birthPlace) { this.birthPlace = birthPlace; }

    public String getNationality() { return nationality; }
    public void setNationality(String nationality) { this.nationality = nationality; }

    public String getPhoto() { return photo; } // Getter nou
    public void setPhoto(String photo) { this.photo = photo; }

    public List<Artwork> getArtworks() { return artworks; }
    public void setArtworks(List<Artwork> artworks) { this.artworks = artworks; }

    public void addArtwork(Artwork artwork) {
        artworks.add(artwork);
        if (artwork.getArtist() != this) {
            artwork.setArtist(this);
        }
    }
}

File: AssignmentMVVM/Model/Artist.cs

namespace AssignmentMVVM.Model
{
    public class Artist
    {
        private string name;
        private DateTime birthDate;
        private string birthPlace;
        private string nationality;
        private string photo;
        private List<Artwork> artworks;


        public Artist()
        {

        }

        public Artist(string name, DateTime birthDate, string birthPlace, string nationality,string photo)
        {
            this.name = name;
            this.birthDate = birthDate;
            this.birthPlace = birthPlace;
            this.nationality = nationality;
            this.photo = photo;
        }

        
        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public DateTime BirthDate
        {
            get { return birthDate; }
            set { birthDate = value; }
        }

        public string BirthPlace
        {
            get { return birthPlace; }
            set { birthPlace = value; }
        }

        public string Nationality
        {
            get { return nationality; }
            set { nationality = value; }
        }

        public string Photo
        {
            get { return photo;  }
            set { photo = value; }
        }
        public List<Artwork> Artworks
        {
            get { return artworks; }
            set { artworks = value; }
        }




    }
}